.github/workflows/pipeline.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.6, 2.7, '3.0', 3.1]
+        ruby: [2.6, 2.7, '3.0', 3.1, 3.2]
```

Comment:
Do we still need to test on version 2.6?

Reply:
We can look to drop Ruby 2.6 support in the future, but I don't think its necessary right now